ldp/nn/lm_config.py

```diff
@@ -44,6 +39,9 @@ class LMConfig(BaseModel):
         default_factory=dict,
         description="Passed as AutoTokenizer.from_pretrained(self.model, **tokenizer_args)",
     )
+    chat_template: str | None = Field(
+        default=None, description="Name of a jinja file defining a chat template."
```

Comment:
What happens if `None`, is there a fallback template? Can you add to the `description` somehow

Reply:
Updated to:
```
    chat_template: str | None = Field(
        default=None,
        description="Name of a jinja file defining a chat template. "
        "Leave as None to not use a chat template.",
    )
    ```